src/main/java/frc/robot/commands/complex/DrivebyAuto.java

```diff
@@ -72,25 +77,28 @@ public void execute() {
 
         pivot.setPosition(targetPivot);
         flywheel.setFlyWheelSpeedMeters(targetFlywheel);
-        swerveSubsystem.drive(new Translation2d(), swerveSubsystem.targetAngleController.calculate(swerveSubsystem.getHeading().getRadians(), targetTurn.getRadians()), true);
+        swerveSubsystem.drive(new Translation2d(), swerveSubsystem.targetAngleController
+                .calculate(swerveSubsystem.getHeading().getRadians(), targetTurn.getRadians()), true);
+
     }
 
     @Override
     public boolean isFinished() {
         double realRotation = swerveSubsystem.getHeading().getRadians();
-        SmartDashboard.putNumber("realFlywheel", realFlywheel);
         SmartDashboard.putNumber("targetFlywheel", targetFlywheel);
+        SmartDashboard.putNumber("realFlywheelTop", flywheel.getFlywheelSpeedMeters());
+        SmartDashboard.putNumber("realFlywheelBottom", flywheel.getAuxiluryFlywheelSpeedMeters());
         SmartDashboard.putNumber("targetPivot", targetPivot);
         SmartDashboard.putNumber("realPivot", realPivot);
         SmartDashboard.putNumber("realRotation", MathUtil.wrapToCircle(realRotation, 2 * Math.PI));
         SmartDashboard.putNumber("targetRotation", MathUtil.wrapToCircle(targetTurn.getRadians(), 2 * Math.PI));
 
         // turn vs pose2d getturn, flywheelreal vs targetflywheel, pivot vs pivot
-        if (MathUtil.isWithinTolerance(realFlywheel, targetFlywheel, 0.5)
-                && MathUtil.isWithinTolerance(realPivot, targetPivot, 0.02)
+        if (MathUtil.isWithinTolerance(realFlywheel, targetFlywheel, 0.08)
+                && MathUtil.isWithinTolerance(realPivot, targetPivot, 0.005)
 
                 && MathUtil.isWithinTolerance(MathUtil.wrapToCircle(realRotation, 2 * Math.PI),
-                        MathUtil.wrapToCircle(targetTurn.getRadians(), 2 * Math.PI), 0.01)) {
+                        MathUtil.wrapToCircle(targetTurn.getRadians(), 2 * Math.PI), 0.05)) {
```

Comment:
try to avoid magic constants by extracting to named constant fields i.e. `private static final double FLYWHEEL_TOLERANCE`. I noticed there is a similar entry in the constants file in #15 so perhaps use that?